metabench.cmake

```diff
@@ -377,6 +377,8 @@ file(WRITE ${CHART_HTML_ERB_PATH}
 "        /* configure the x axis */                                          \n"
 "        var xAxis = chart.xAxis || {};                                      \n"
 "        delete(chart.xAxis);                                                \n"
+"        if(typeof xAxis.axisLabel == 'undefined')                           \n"
+"          xAxis.axisLabel = 'Number of Elements'                            \n"
```

Comment:
The X axis is not always a number of elements. For example, for the `at` benchmark, it's actually the index of the element we're accessing. (Granted, it's also the number of elements in the sequence for now but that will change since this benchmark is broken as is).